testbed/lib/main.dart

```diff
@@ -321,7 +321,7 @@ class FileChooserTestWidget extends StatelessWidget {
           child: const Text('OPEN'),
           onPressed: () async {
             String initialDirectory;
-            if (Platform.isMacOS || Platform.isWindows) {
+            if (Platform.isMacOS || Platform.isWindows || Platform.isLinux) {
```

Comment:
These are the only platforms this repository is concerned with, so you can just remove the condition.